download/bindings-fr.php

```diff
@@ -178,6 +178,12 @@
             <td>2.5.1</td>
             <td>Guillaume Derex</td>
         </tr>
+        <tr>
+            <th rowspan="1">Node.js</th>
+            <td><a href="https://github.com/XadillaX/node-sfml" class="website">sfml.js</a></td>
```

Comment:
Wouldn't node-sfml make more sense instead of sfml.js given that the project is called node-sfml?

Reply:
It's package name in npm is [sfml.js](https://www.npmjs.com/package/sfml.js).